assets/css/style.css

```diff
@@ -57,13 +57,23 @@
   background-color: #26a65b;
 }
 
+.main-nav-link-active {
+  width: 100%;
+  border-bottom: 4px solid #26a65b;
```

Comment:
Do not use directly the color #26a65b, but create a CSS variable with the value #26a65b and use that variable across all places. 

We do it this way to follow the [DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).